### ISSUE TITLE
Risk of NullPointerException Due to Use of Boxed Boolean in toPath Method

### ISSUE THREAD

**Issue by tanmaysharma2001** [cqfn/jpeek#554]
I've noticed a potential issue in the FileTarget.java file where a boxed Boolean is used directly in a boolean expression. This occurs in the toPath method, specifically at [this line](https://github.com/cqfn/jpeek/blob/master/src/main/java/org/jpeek/FileTarget.java#L63). The use of a boxed Boolean (this.overwrite) in a conditional check might lead to a NullPointerException if the Boolean object is null.

### Potential Impact

In cases where this.overwrite is null, attempting to evaluate it in a boolean context (if (this.overwrite)) will throw a NullPointerException, leading to unexpected crashes or behavior.

A much compliant solution here will be to use a Primitive Boolean Expression.

@yegor256 Can you please approve this issue?

**Comment by yegor256** [comment]
@tanmaysharma2001 yes, we can change the type of the attribute to simply `boolean`
